doc/source/user_guide/hpc.rst

```diff
@@ -0,0 +1,16 @@
+
+.. _ref_hpc:
+
+
+===============================
+High performance clusters (HPC)
+===============================
+
+In this page, an overview on how to use PyMAPDL in HPC clusters is presented.
+At the moment, only SLURM scheduler is considered.
+However, many of the assumptions for this scheduler might apply to other schedulers
+like PBS, SGE, or LSF.
```

Comment:
```suggestion
This page provides an overview on how to use PyMAPDL in HPC clusters.
While it only considers the SLURM scheduler, many of the assumptions for this scheduler might apply to other schedulers, such as PBS, SGE, or LSF.
```
Documentation should be timeless, always reflecting the current status. Avoid terms like "At the moment" and words like presently and currently. It should also be written in active voice.